### ISSUE TITLE
Error Google drive

### ISSUE THREAD

**Issue by hunchoap** [floccusaddon/floccus#1644]
### Which version of floccus are you using?

5.2.0

### How many bookmarks do you have, roughly?

100

### Are you using other means to sync bookmarks in parallel to floccus?

no

### Sync method

Google Drive

### Describe the Bug

![CleanShot 2024-06-12 at 8  27 54@2x](https://github.com/floccusaddon/floccus/assets/171800885/ac298b39-e4d7-42f5-9efc-26e9352fd180)

### Expected Behavior

/ 

### To Reproduce

Normal setup but choose google drive 

### Debug log provided

- [X] I have provided a debug log file

**Comment by marcelklehr** [comment]
This happens sometimes. I don't know why, though. Perhaps try again after a while?

**Comment by hunchoap** [comment]
I tried like 1 month ago and still the same issue, never worked

**Comment by marcelklehr** [comment]
This is a duplicate of #1562 I believe
